### PR TITLE
Docstrings

### PR DESCRIPTION
**Pull request by franramirez688**
Changelog: omit
Docs: omit

### CODE REVIEW

**Review by memsharded — APPROVED** [review]
But the ``conf`` was not just a docstring, how test could pass without it? I think because it was just a unit test that is not running validation of the ``conf`` inputs

**Comment by franramirez688** [comment]
> But the `conf` was not just a docstring, how test could pass without it? I think because it was just a unit test that is not running validation of the `conf` inputs

It's only an order change. It was completely right.

File: conans/model/conf.py [hunk 1]
```diff
@@ -108,7 +108,6 @@
     "tools.apple:enable_visibility": "(boolean) Enable/Disable Visibility Apple Clang flags",
     "tools.env.virtualenv:powershell": "If it is set to True it will generate powershell launchers if os=Windows",
     # Compilers/Flags configurations
-    "tools.build.cross_building:cross_build": "Decides whether cross-building or not regardless of arch/OS settings. Used by 'conan.tools.build.cross_building' function.",
```
Comment (franramirez688) [inline conans/model/conf.py:111]: @memsharded It was here. Now, it's above. Otherwise, the test would fail.

Comment (memsharded) [reply conans/model/conf.py:111]: Oh, you are right! Sorry I didn't see it.

**Pull request merged**